server/src/main/java/org/apache/cloudstack/outofbandmanagement/PowerOperationTask.java

```diff
@@ -43,8 +53,26 @@ public void run() {
         try {
             service.executePowerOperation(host, powerOperation, null);
         } catch (Exception e) {
-            LOG.warn(String.format("Out-of-band management background task operation=%s for host id=%d failed with: %s",
-                    powerOperation.name(), host.getId(), e.getMessage()));
+            LOG.warn(String.format("Out-of-band management background task operation=%s for host id=%d name=%s failed with: %s",
+                    powerOperation.name(), host.getId(), host.getName(), e.getMessage()));
```

Comment:
In general in CloudStack we try to not log the 'hostid' as it doesn't say anything. The ID can only be fetched from the DB and not from the API and thus UI.

Reply:
Makes sense, I am going to keep only the host name then.